SDK/AppCenter/Microsoft.AppCenter.Android.Bindings/Microsoft.AppCenter.Android.Bindings.csproj

```diff
@@ -16,6 +16,15 @@
     <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
     <AndroidTlsProvider>
     </AndroidTlsProvider>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
```

Comment:
Visual Studio for Mac will insist on copying the root commands to each configuration everytime we build, so removed at root to avoid having 3 times the commands instead of 2...